Mathlib/RingTheory/Artinian.lean

```diff
@@ -304,24 +304,6 @@ end IsArtinian
 
 end CommRing
 
--- TODO: Prove this for artinian modules
--- /--
--- If `M ⊕ N` embeds into `M`, for `M` noetherian over `R`, then `N` is trivial.
--- -/
--- universe w
--- variables {N : Type w} [add_comm_group N] [module R N]
--- noncomputable def is_noetherian.equiv_punit_of_prod_injective [is_noetherian R M]
```

Comment:
This is `IsNoetherian.equivPUnitOfProdInjective` in `Mathlib/RingTheory/Noetherian.lean` below